internal/handler/msg_serverstatus.go

```diff
@@ -69,6 +70,41 @@ func (h *Handler) msgServerStatus(connCtx context.Context, req *middleware.Reque
 		}
 	}
 
+	// Add CommandFeatureCounterStats as Prometheus metrics
+	featureCounterStatsDoc := wirebson.MakeDocument(0)
+
+	// Try to get feature counter stats, but don't fail if the function doesn't exist
+	err = h.Pool.WithConn(func(conn *pgx.Conn) error {
+		// Query for all feature counter stats
+		rows, queryErr := conn.Query(connCtx,
+			`SELECT feature_name, usage_count FROM documentdb_api_internal.command_feature_counter_stats($1)`,
+			false,
+		)
+		if queryErr != nil {
+			// If the function doesn't exist or other error, just log and continue
```

Comment:
I'm not seeing log :thinking:? More importantly, ignoring error makes testing this feature hard, because it's difficult to notice if it stops working. Why can't we fail serverStatus if feature counter stats fails?